scarb/src/bin/scarb/args.rs

```diff
@@ -228,6 +228,14 @@ pub struct ExpandArgs {
     #[command(flatten)]
     pub features: FeaturesSpec,
 
+    /// Specify the target to expand by target kind.
+    #[arg(long)]
+    pub target_kind: Option<String>,
+
+    /// Specify the target to expand by target name.
+    #[arg(long)]
+    pub target_name: Option<String>,
+
```

Comment:
noob question: is it possible someone sets a target_name and a target_kind that are incompatible with each other?

Reply:
Ah ok, so setting kind will result in extending all applicable targets of this kind, right? whereas setting the name will result in extending only this specific target? 

Makes sense, thanks for clarification :+1:

Reply:
> setting kind will result in extending all applicable targets of this kind

Not really, right now this would be the first one that matches.

Reply:
discussed offline